### ISSUE TITLE
unable to ban relative imports

### ISSUE THREAD

**Issue by zkurtz** [astral-sh/ruff#14374]
Here's an entire minimal repo dedicated to illustrating the issue: https://github.com/zkurtz/tmp_debug_ruff

My pyproject.toml includes the snippet provided [in the docs]( https://docs.astral.sh/ruff/settings/#lint_flake8-tidy-imports_ban-relative-imports):
```
[tool.ruff.lint.flake8-tidy-imports]
ban-relative-imports = "all"
```

My repo includes a relative import: `from . import blah`.

But running `ruff check .` in my repo says all checks passed, and `ruff check --fix` makes no changes. My interpretation of the rule was that it either should throw an error or fix the import to `from src.import blah`.

My ruff version is 0.7.4.

**Comment by charliermarsh** [comment]
Thanks for the repro. You need to enable the rule -- it's not enabled by default:

```shell
❯ uvx ruff check --select TID
src/__init__.py:1:1: TID252 Prefer absolute imports over relative imports
  |
1 | from . import blah
  | ^^^^^^^^^^^^^^^^^^ TID252
2 | # expecting ruff to require this to be `from src import blah`
  |
  = help: Replace relative imports with absolute imports

Found 1 error.
No fixes available (1 hidden fix can be enabled with the `--unsafe-fixes` option).
```

You add this to your `pyproject.toml`, as an example:

```toml
[tool.ruff.lint]
extend-select = ["TID"]
```

**Comment by zkurtz** [comment]
Success, thank you! My only follow up question is whether the docs could be clarified; it's still not obvious to me where the docs imply this solution.

**Comment by charliermarsh** [comment]
Sweet. There's some information on rule selection here: https://docs.astral.sh/ruff/linter/#rule-selection
